verrazzano/filebeat/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2020, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-FROM container-registry.oracle.com/os/oraclelinux:7.8 as build_base
+FROM container-registry.oracle.com/os/oraclelinux:7.8@sha256:46fc083cf0250ed5260fa6fe822d7d4c139ca1f7fc38e4a17ba662464bd1df4a as build_base
```

Comment:
Is it necessary to build on 7.8?  Would 7-slim work?

Reply:
not sure but i am not changing the images, just the tags

Reply:
not changing the image, just the tag